packages/connect-examples/native-android-example/web/index.js

```
@@ -0,0 +1,227 @@
+    const plugin = createLowlevelPlugin();
+
+    try {
+      await HardwareSDK.init(settings, undefined, plugin);
+      console.log("HardwareSDK init success");
+      isInitialized = true;
+      resolve(HardwareSDK);
+      listenHardwareEvent(HardwareSDK);
+    } catch (e) {
+      reject(e);
+    }
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)